include/swift/AST/DiagnosticsSema.def

```diff
@@ -3685,6 +3685,9 @@ ERROR(inconsistent_implicit_access_level_on_import,none,
       "ambiguous implicit access level for import of %0; it is imported as "
       "'%select{private|fileprivate|internal|package|public|%error}1' elsewhere",
       (Identifier, AccessLevel))
+NOTE(inconsistent_implicit_access_level_on_import_silence,none,
+      "silence these warnings by adopting 'InternalImportsByDefault'",
```

Comment:
It may not be clear what `adopting 'InternalImportsByDefault'` means; how about:
```suggestion
      "silence these warnings by adopting the upcoming feature 'InternalImportsByDefault'",
```

Reply:
Good idea, it's applied. In all cases I expect people will have to search for the meaning of this feature and how to enable it, and this clarifies what to look for.